var/spack/repos/builtin/packages/systemd/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Systemd(AutotoolsPackage):
+    """systemd is a suite of basic building blocks for a Linux system.
+    It provides a system and service manager that runs as PID 1 and
+    starts the rest of the system."""
+
+    homepage = "https://systemd.io/"
+    url = "https://github.com/systemd/systemd/archive/refs/tags/v255.tar.gz"
+    license("GPL-2.0-only")
+
+    version("255", sha256="28854ffb2cb5f9e07fcbdbaf1e03a80b3462a12edeef84893ca2f37b22e4491e")
+
+    depends_on("meson@0.60.0", type="build")
+    depends_on("ninja", type="build")
+    depends_on("py-jinja2", type="build")
+    depends_on("util-linux@2.30:")  # libmount
+    depends_on("gperf")
+    depends_on("libcap")
+    depends_on("pcre2")
+    depends_on("libdwarf")
+    depends_on("libidn2")
+    depends_on("openssl")
+    depends_on("pkgconfig")
+
+    conflicts("%gcc@:8.4")
+    conflicts("%clang@:7")
+
+    def configure_args(self):
+        return ["--prefix=/"]
+
+    def install(self, spec, prefix):
+        make(f"DESTDIR={prefix}", "install")
```

Comment:
I guess we only want systemd for its libraries, right? Is there a way of disabling everything else since it probably won't work anyway?

Reply:
That's correct - I only need libsystemd.so do fulfill the dependency that geopm-service has.  I'll look at disabling all that I do not need.